v1/api/swagger/components/schemas/OSServiceImageDownloadHealth.yaml

```diff
@@ -4,8 +4,22 @@ required:
   - DownloadHealthSummary
 properties:
   DetailedDownloadHealth:
+    type: object
+    description: This is a map of controllers ID to detailed download health info.
     $ref: OSServiceImageDetailedDownloadHealth.yaml
+    example: 
+      046b6c7f-0b8a-43b9-b35d-6489e6daee91:
+        Summary: OK
+        Reason: ''
+        Changed: 2019-06-27T22:59:01.661Z
+      91eec8b5-6460-4532-aad1-47b983edb63c:
+        Summary: OK
```

Comment:
Same as above, example of non-healthy status? Also, why is this repeated? Doesn't the $ref automatically pull in the example from that file?